src/main/java/io/vertx/core/http/impl/Http2ServerRequestImpl.java

```diff
@@ -479,7 +479,11 @@ public HttpServerRequest setExpectMultipart(boolean expect) {
                   uri);
               req.headers().add(HttpHeaderNames.CONTENT_TYPE, contentType);
               postRequestDecoder = new HttpPostRequestDecoder(new NettyFileUploadDataFactory(context, this, () -> uploadHandler), req);
+            } else {
+                throw new IllegalStateException("Invalid content type or http method. Content type should be "+HttpHeaderValues.MULTIPART_FORM_DATA.toString()+" or "+HttpHeaderValues.APPLICATION_X_WWW_FORM_URLENCODED+" and http method should be "+ io.netty.handler.codec.http.HttpMethod.POST.toString()+", " + io.netty.handler.codec.http.HttpMethod.PUT.toString() + ", "+ io.netty.handler.codec.http.HttpMethod.PATCH.toString()+" or "+io.netty.handler.codec.http.HttpMethod.DELETE.toString()+".");
```

Comment:
can you just write POST, PUT, PATCH, DELETE ? that's easier to read and those are well known constants that don't need to use Java constants.